x/staking/module.go

```diff
@@ -85,7 +85,9 @@ func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Rout
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the staking module.
 func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
 	// TODO : uncomment below line
-	types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
```

Comment:
same as above

Reply:
In this case, for returning the error, I will have to change the function signature which might not be a good idea.
1. Would it be ok to just do
`_ = types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))`
like what we did for `cmd.MarkFlagRequired`
Or
2. panic as this is in module.go
@anilCSE @jdkanani

Reply:
Implemented (1) as of now. Please let me know if I need to change.